### ISSUE TITLE
The solution to migrate code to compatible with Tensorflow1.0.1

### ISSUE THREAD

**Issue by loveJasmine** [carpedm20/deep-rl-tensorflow#27]
diff --git a/networks/layers.py b/networks/layers.py
index 86d6052..5c8e329 100644
--- a/networks/layers.py
+++ b/networks/layers.py
@@ -7,7 +7,7 @@ def conv2d(x,
            kernel_size,
            stride,
            weights_initializer=tf.contrib.layers.xavier_initializer(),
-           biases_initializer=tf.zeros_initializer,
+           biases_initializer=tf.zeros_initializer(),
            activation_fn=tf.nn.relu,
            data_format='NHWC',
            padding='VALID',
diff --git a/networks/mlp.py b/networks/mlp.py
index 9c9e58f..8f595f8 100644
--- a/networks/mlp.py
+++ b/networks/mlp.py
@@ -12,7 +12,7 @@ class MLPSmall(Network):
                trainable=True,
                batch_size=None,
                weights_initializer=initializers.xavier_initializer(),
-               biases_initializer=tf.zeros_initializer,
+               biases_initializer=tf.zeros_initializer(),
                hidden_activation_fn=tf.nn.relu,
                output_activation_fn=None,
                hidden_sizes=[50, 50, 50],
